data_processors/lims/lambdas/google_lims.py

```diff
@@ -26,17 +25,21 @@
 logger.setLevel(logging.INFO)
 
 
-def scheduled_update_handler(event, context) -> Dict[str, int]:
-    """
-    Handler for LIMS update by reading the designated Spreadsheet file from Google Drive.
-    Can be hit by Cron like job scheduler. It uses EventBridge on default EventBus with cron(0 12 * * ? *). See REF.
-    Can be also invoked directly:
-        aws lambda invoke --function-name data-portal-api-[dev|prod]-lims_scheduled_update_processor
+def _halt(msg):
+    logger.error(msg)
+    return {
+        'message': msg
+    }
+
 
-    REF:
-    https://docs.aws.amazon.com/eventbridge/latest/userguide/scheduled-events.html
-    https://docs.aws.amazon.com/eventbridge/latest/userguide/event-types.html#schedule-event-type
-    https://docs.aws.amazon.com/eventbridge/latest/userguide/run-lambda-schedule.html
+def scheduled_update_handler(event, context):
```

Comment:
Just wondering so the truncate here, will always either be a warning or a `NotImplementedError`?

Reply:
Yep; it throws NIE exception when user is attempting to truncate true. That's the intention. To open discussion whether should we ever need to have a use case for this. Otherwise, it is never called; truncate false; hence update & merge for exisiting records; i.e. the current ops requirement.